Justfile

```diff
@@ -148,8 +130,6 @@ build $image="aurora" $tag="latest" $flavor="main" rechunk="0" ghcr="0" pipeline
     # AKMODS Flavor and Kernel Version
     if [[ "${flavor}" =~ hwe ]]; then
         akmods_flavor="bazzite"
-    elif [[ "${tag}" =~ stable ]]; then
-        akmods_flavor="coreos-stable"
```

Comment:
I believe these lines were not meant to be removed. I tracked down the discrepancy in the NVIDIA kernel module version vs the NVIDIA software stack versions (#148) to this removal. We are using the main kernel for akmods for `stable-daily` when we should be using the stable (coreos) kernel.